catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/RemoteDeleteOperationsTest.java

```diff
@@ -268,8 +268,8 @@ private void assertEqualMetacards(String reason, List<Metacard> metacardList1,
                 .map(Metacard::getTitle)
                 .collect(Collectors.toList());
 
-        assertThat(reason, idList1, containsInAnyOrder(idList2));
-        assertThat(reason, titleList1, containsInAnyOrder(titleList2));
+        assertThat(reason, idList1, containsInAnyOrder(idList2.toArray()));
+        assertThat(reason, titleList1, containsInAnyOrder(titleList2.toArray()));
```

Comment:
That's simply the nature of asserting over a collection. It expects a list in the array format or else it complains (which it did). Is this a show-stopper or just a general question?